.github/workflows/node.js.yml

```
@@ -56,22 +63,34 @@ jobs:
+    runs-on: ubuntu-latest
+    needs: build
+    if: |
+      github.event_name == 'pull_request' &&
+      contains(github.actor, 'dependabot')
+
+    permissions:
+      pull-requests: write
 
+    steps:
       - name: Post comment
         uses: actions/github-script@v4
```

actionlint (1.7.12)

[error] the runner of "actions/github-script@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)